packages/design-toolkit/src/components/coordinate-field/styles.module.css

```diff
@@ -42,11 +42,27 @@
   }
 
   .input {
-    @apply gap-xs flex flex-1 items-center;
+    @apply flex items-center;
+
+    div {
+      @apply w-full;
+    }
```

Comment:
double checking you want this applied to every child div in future or if you want it just for the segment group?

if just the group, consider a more specific classname like .segment-group here